evennia/utils/valid.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from evennia.utils.utils import callables_from_module
+
+
+class ValidHandler(object):
```

Comment:
As you pointed out, this is not really handler in the Evennia sense of the word. I don't think "helper" is a good word either though - it's just a container that preloads resources from a module. "Container" may be a pretty good term actually. 

I think overall though, it would be preferable if resources on these containers could be accessed as properties rather than with dict-like operations - the latter is quite different from how other Evennia APIs operate.